server/index.js

```diff
@@ -0,0 +1,14 @@
+const config = require('./config/config');
+const routes = require('./routes');
+const express = require('express');
+const http = require('http');
+
+const app = express();
+app.server = http.createServer();
+app.disable('x-powered-by');
```

Comment:
what do

Reply:
For some reason express apps send back an 'X-Powered-By' header in the response, and I've seen it said that it's safer to disable this so that any potential attackers don't know that the server is running on express.

The thought is that if suddenly tomorrow there's some massive express bug that's discovered and we aren't able to update or whatever to address it in time, attackers could see that we're running an express server and know that we're potentially vulnerable to whatever the issue is.

Not a huge concern, but something I had seen and thought might be worth including